iOS_Swift/NOTIFICATIONS.md

```diff
@@ -0,0 +1,35 @@
+# iOS Notifications
+
+### What are they?
+
+Notifications are a way for your app to reach out to users while your app is not running. This may be just to notify them of something, or to bring them back into your app to perform some action.
```

Comment:
:+1: I learned something :smile: I never even considered that local and push notifications were different!